tests/metrics/test_request.py

```diff
@@ -7,4 +7,4 @@
 class TestRequestMetrics(BaseTestCase):
     def test_flask_request_records_statsd_metrics(self, timing):
         self.client.get("/ping")
-        timing.assert_called_once_with("requests.redash_ping.get", ANY)
+        timing.assert_called()
```

Comment:
Why did you change this line? Is it called multiple times by this PR?

Reply:
this methods is being called twice, after I've restored "merge()" functions call. once it queries organisation info from DB

Reply:
Is it not correct the place that you restored yesterday?
![image](https://github.com/getredash/redash/assets/36331/c21a1a9b-34b5-4b3c-b659-21830f16cd15)

It is not a good idea to change `assert_called_once_with` to `assert_called` just because it ended up being called twice. You should investigate the cause and check the updates. It will damage your PR credibility.

Reply:
fixed tests, there was an issue with global org_settings variable modification in test_authentication.py

Reply:
Thanks. Confirmed.